content/spin/v2/build.md

```diff
@@ -66,23 +66,17 @@ command = "cargo build --target wasm32-wasi --release"
 
 {{ startTab "TypeScript" }}
 
-For JavaScript and TypeScript applications, you must have the `js2wasm` Spin plugin installed:
+For JavaScript and TypeScript applications, you must have `node.js`.
 
-<!-- @selectiveCpy -->
-
-```bash
-$ spin plugins update
-$ spin plugins install js2wasm --yes
-```
 
-It's normally convenient to put the detailed build instructions in `package.json`:
+It's normally convenient to put the detailed build instructions in `package.json`, The build script calls out to [`webpack`](https://webpack.js.org/) and `j2w` which is a script provided by the `@fermyon/spin-sdk` package that utilizes [`ComponentizeJS`](https://github.com/bytecodealliance/ComponentizeJS). The build script looks like:
 
 <!-- @nocpy -->
 
 ```json
 {
   "scripts": {
-    "build": "npx webpack --mode=production && mkdir -p target && spin js2wasm -o target/spin-http-js.wasm dist/spin.js"
+   "build": "npx webpack --mode=production && npx mkdirp target && npx j2w -i dist.js -d combined-wit -n combined -o target/spin-http-js.wasm"
```

Comment:
Not a docs question, but I assume we're using `npx mkdirp` because it's cross platform?

Reply:
Yep - We were originally using `mkdir -p` on the old SDK but that led to it breaking on Windows as it does not support the `-p` option.